operations/loyaltymemberships.md

```diff
@@ -65,6 +70,14 @@ Note this operation uses [Pagination](../guidelines/pagination.md).
 * `Active` - active records (the validity might be restricted by another parameter i.e. interval).
 * `Deleted`- deleted records.
 
+#### Loyalty membership state
+
+* `New`
+* `Pending`
+* `Enrolled`
+* `Canceled`
+* `Declined`
```

Comment:
same story is this enum static?

Reply:
That's the plan yes.

Reply:
Updated it with `* ...` after our discussion in slack 👍🏻